doc/en/example/simple.rst

```diff
@@ -131,21 +131,19 @@ line option to control skipping of ``slow`` marked tests::
         parser.addoption("--runslow", action="store_true",
             help="run slow tests")
 
-    def pytest_runtest_setup(item):
-        if 'slow' in item.keywords and not item.config.getoption("--runslow"):
-            pytest.skip("need --runslow option to run")
-
 We can now write a test module like this::
 
     # content of test_module.py
 
     import pytest
-    slow = pytest.mark.slow
```

Comment:
just a sidenote: if for some reason a test file with the above definition would be read at conftest.py-reading time, it would not work because command line options only get parsed after conftest.py files are read.  You can wrap the expression into a string, though, ``'not config.getoption("--runslow")'`` which will make it get evaluated at test-run-time IIRC.